ci/task_e2e_tests.yaml

```diff
@@ -3,8 +3,8 @@ platform: 'linux'
 image_resource:
   type: docker-image
   source:
-    repository: golang
-    tag: 1.9.2-alpine3.6
+    repository: sapcc/kubernikus-tests
+    tag: latest
```

Comment:
Maybe this should be a specific version so that we are sure what is running...

Reply:
@databus23 I changed this.